src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { ApiError } = require('../exceptions/ApiError.js');
+const jwtService = require('../services/jwt.service.js');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'];
```

Comment:
it is more common practice to start with upper case:
```suggestion
  const authHeader = req.headers['Authorization'];
```

Reply:
Understand. 
But you have to fix it in the example files, because that's what your colleagues teach us.
https://github.com/mate-academy/nodejs-auth-api-done/blob/master/src/middlewares/authMiddleware.js